pkg/persistence/config/loader/config_loader_test.go

```diff
@@ -720,6 +721,91 @@ configs:
 				},
 			},
 		},
+		{
+			name:             "loads settings 2.0 config with all properties and all-users permission with FF on",
+			envVars:          map[string]string{featureflags.AccessControlSettings.EnvName(): "true"},
+			filePathArgument: "test-file.yaml",
+			filePathOnDisk:   "test-file.yaml",
+			fileContentOnDisk: `
+configs:
+- id: profile-id
+  config:
+    name: 'Star Trek > Star Wars'
+    template: 'profile.json'
+    originObjectId: origin-object-id
+  type:
+    settings:
+      schema: 'builtin:profile.test'
+      schemaVersion: '1.0'
+      scope: 'tenant'
+      permissions:
+        all-users: 'read'`,
```

Comment:
Is `all-users` meant to be kebab case? I see we already have `originObjectId`, `insertAfter` and `schemaVersion`...

Reply:
Agree, i will update it and the story.